README.md

```diff
@@ -6,82 +6,82 @@ OpenShift Application Platform
 [![Travis](https://travis-ci.org/openshift/origin.svg?branch=master)](https://travis-ci.org/openshift/origin)
 [![Jenkins](https://ci.openshift.redhat.com/jenkins/buildStatus/icon?job=devenv_ami)](https://ci.openshift.redhat.com/jenkins/job/devenv_ami/)
 [![Join the chat at freenode:openshift-dev](https://img.shields.io/badge/irc-freenode%3A%20%23openshift--dev-blue.svg)](http://webchat.freenode.net/?channels=%23openshift-dev)
+[![Licensed under Apache License version 2.0](https://img.shields.io/github/license/openshift/origin.svg?maxAge=2592000)](https://www.apache.org/licenses/LICENSE-2.0)
 
-OpenShift Origin is a distribution of [Kubernetes](https://kubernetes.io) optimized for continuous application development and multi-tenant deployment.  Origin adds developer and operations-centric tools on top of Kubernetes to enable rapid application development, easy deployment and scaling, and long-term lifecycle maintenance for small and large teams.
+***OpenShift Origin*** is a distribution of [Kubernetes](https://kubernetes.io) optimized for continuous application development and multi-tenant deployment.  Origin enables teams of all sizes to quickly develop, deploy, and scale applications, reducing the length of development cycles and operational effort.
```

Comment:
@danmcp Thanks for the review. "OpenShift Origin is a distribution of Kubernetes" isn't sufficient to indicate that we extend Kube? Would it be clearer to rephrase as:

> Origin extends Kubernetes, enabling teams of all sizes to quickly develop, deploy, and scale applications, reducing the length of development cycles and operational effort.

I just worry that this makes the sentence too complicated, since we're trying to convey three separate things:

1. that we're extending Kube;
2. that we have features to make development/deployment/scaling easier;
3. that we anticipate benefits like faster time-to-market/quicker development cycles, and ease of operation

Or do you have another phrasing in mind that might be clearer? I'm just trying to avoid being too redundant and buzzwordy; the first sentence is our chance to articulate our value to users.